ChustaSoft.Common.AspNet/ChustaSoft.Common.AspNet.csproj

```diff
@@ -9,9 +9,9 @@
 		<Authors>Xelit3</Authors>
 		<Company>ChustaSoft</Company>
 		<RestoreProjectStyle>PackageReference</RestoreProjectStyle>
-		<Version>2.4.0</Version>
-		<FileVersion>2.4.0</FileVersion>
-		<AssemblyVersion>2.4.0</AssemblyVersion>
+		<Version>2.5.0</Version>
```

Comment:
We can avoid by the moment the version increase after evth in the current improvents is deployed.